Clarify cli acquisition milestones and provide ecosystem context

baronfel opened the pull request:
We want to make it clear that the public preview phase is still treating the tool as an experiment - we want to collect feedback and data to turn into a "should we make this a fully-supported product" decision through the public preview period.

Then, based on that go/no-go decision with management, we have SDL and fit-and-finish requirements to go implement.

from marcpopMSFT:
Added some ecosystem context that clarifying how other tools work today and what customers are gravitating towards.

baronfel committed:
Add some details to milestone descriptions

We want to make it clear that the public preview phase is still treating the tool as an experiment - we want to collect feedback and data to turn into a "should we make this a fully-supported product" decision through the public preview period.

Then, based on that go/no-go decision with management, we have SDL and fit-and-finish requirements to go implement.

baronfel committed:
Update accepted/2025/cli-acquisition-tool.md

Co-authored-by: Rolf Bjarne Kvinge <rolf@xamarin.com>

marcpopMSFT committed:
Adding summary ecosystem level detail.

jaredpar commented on accepted/2025/cli-acquisition-tool.md:
```diff
@@ -184,6 +184,36 @@ This command will check documented, signed release manifests for `dnup` and down
+*	**Go and Java:** Language runtimes often installed centrally, but developers increasingly use version managers or user-scoped installs to avoid admin rights.
+*	**PATH-based discovery** is universal across ecosystems.
+*	**Isolation mechanisms** (virtual environments, local installs) are highly valued for avoiding dependency conflicts.
+
```
Feel like we need to pick one or two of these and dive deeper. Basically what happens with nvm when there is a global install, how do they recommend it get handled and what is the user experience around it? Think knowing that for a few others helps put into context some of the challenges that we have here.

Copilot committed:
Initial plan

Copilot committed:
Expand Ecosystem Context section with nvm and rustup deep dives

Co-authored-by: baronfel <573979+baronfel@users.noreply.github.com>

baronfel committed:
Merge pull request #360 from dotnet/copilot/expand-ecosystem-context-nvm-rustup

Expand Ecosystem Context with nvm/rustup deep dives and command comparisons

nagilson committed:
Add 2026 Folder, Move `dotnetup` proposal

Move dotnetup proposal from @baronfel into 2026 folder. Doing this in the same commit to cause detection of the new folder

nagilson committed:
Add comparison for nvm-windows and dotnetup

In this PR, I add onto the proposal from @baronfel to include a direct comparison, showing some of what I researched about `nvm` and  how  we thought about it when designing  `dotnetup`.

Some of this is pretty specific into technical detail and I want to avoid this document becoming huge and unreviewable. @baronfel's document also is a larger picture sell rather than a dive into the specifics, however I thought it would be helpful to have this information here. We can close these changes if we disagree and move it elsewhere.

I tried to briefly touch on the topics without duplicating what will go into `dotnet/sdk/documentation/dotnetup` -  the `hive` defaults and `path` handling alone will be their own documents. It can take several pages to explain.

nagilson committed:
Modify index.md to include moved sub dotnetup folder

nagilson committed:
Merge branch 'baronfel-patch-1' into nagilson-add-comparison-nvm-dotnetup-to-baronfel-spec

nagilson committed:
Unix Comparison

More of this is generated text, but I've modified it and done some fact checking.

nagilson committed:
fix terminology

Co-authored-by: Jan Kotas <jkotas@microsoft.com>

nagilson committed:
remove

nagilson committed:
improve url specificity

nagilson committed:
Merge pull request #361 from dotnet/nagilson-add-comparison-nvm-dotnetup-to-baronfel-spec

Add comparison for nvm and dotnetup
